infrastructure/monitoring/elastalert/elastalert.yaml

```diff
@@ -16,8 +16,6 @@ buffer_time:
 
 es_host: elasticsearch
 es_port: 9200
-es_username: '{{ES_USERNAME}}'
-es_password: '{{ES_PASSWORD}}'
```

Comment:
These are probably needed for Elastalert to be able to connect to ES?

Reply:
Ah, these can indeed be removed as environment variables passed down to Elastalert overwrite and have been overwriting these for a long time